addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -617,7 +617,6 @@ blockquote {
         $-bg-color: o-color($-bg);
         @include o-bg-color($-bg-color, o-color($-text), $important: false, $yiq-min-opacity-threshold: 0);
         @if $-bg-gradient {
-            background-color: transparent;
```

Comment:
Oh !!! Actually, the whole `$-bg-gradient` cannot be inside this file: it cannot use `o-website-value` to get its value.
So, better remove everything about it here - and reintroduce it inside an SCSS of website.

Reply:
@bso-odoo is `editor.scss` is correct file for this changes to move in website ? As i'm confused where to put this scss in website.

Reply:
@visp-odoo As far as I can tell by reading website's manifest file, `editor.scss` is only included in backend assets. I think we need these `o_cc*` styles to be applied when displaying frontend pages. So, I'd rather go for `website.scss` which is included in the frontend.
Pinging @qsm-odoo to be sure I am not leading you in the wrong direction.

Reply:
@visp-odoo
- The fact that .o_ccX rules are in web_editor.common.scss, is probably a mistake on its own. But fixing that should be part of a better review of "what is where" in web_editor & website. This can wait. Better have it there for no reason that moved somewhere else and break features I don't know about / remember. The point of it being in common was probably to have it shared with mass mailing at some point... but... don't know.
- However, this file cannot mention website stuff indeed (cc @bvr-odoo)... and from what I understand this was merged in 17.1, so your PR here should target 17.1 as well, not master?
- Yes, it should probably be in website.scss as @bso-odoo explained. Noting to be confused about: make sure the feature that introduced these lines (https://github.com/odoo/odoo/commit/48b74a48d2740f9f845ce03ef646e1eaf6bb2a9c) still works as it should after you move the lines :+1: